dspace-api/src/main/java/org/dspace/app/requestitem/RequestItemAuthorExtractor.java

```diff
@@ -19,15 +20,15 @@
  * @author Andrea Bollini
  */
 public interface RequestItemAuthorExtractor {
-
     /**
-     * Retrieve the auhtor to contact for a request copy of the give item.
+     * Retrieve the author to contact for a request copy of the give item.
      *
      * @param context DSpace context object
      * @param item item to request
      * @return An object containing name an email address to send the request to
```

Comment:
this need to be adjusted to the list return. Is null allowed or we require an empty list?

Reply:
Javadoc has been updated.  Method has been annotated `@NonNull`.